### ISSUE TITLE
Request / Suggestion: add "strategic launch detection" on-screen notify.

### ISSUE THREAD

**Issue by KarlMarx420faf** [FAForever/fa#4684]
**Describe the feature**
A visual indicator for active strategic launches. For players without sound or hard of hearing, this would help maintain parity during multiplayer matches. Would also improve general QoL if players prefer sound off for other reasons. 

**Expected behavior**
Similar to how notify allows players to automatically announce upgrades in chat, or how casters have on-screen "X built an experimental" messages, a "Strategic Launch Detected" text would appear when enemies (or allies) launch a nuke. On the forums, someone suggested using red or green icons instead of text to indicate the nuke launches -- this is an interesting approach because if many nukes are launched at once it might become cumbersome. At least off the cuff, the options that make the most sense to me are:

1. One on-screen red or green nuke icon per enemy or allied nuke airborne. They can be static icons along the top of the screen.
2. Text in the chatbox similar to how players automatically announce tech upgrades through notify.
3. Text on the top of the screen similar to how casters like gyle or willlow have it in replays.

**Screenshots**
An example from user thecore on FAF forums. 

[https://forum.faforever.com/assets/uploads/files/1675934286053-untitled-1.jpg](url)

**Additional context**

I play on macbook pro with an i9. I tried every trick on the forums to get sound working including an external sound card but couldn't get sound to work without the game crashing. I settled for playing without sound, but run into parity issues when I am unaware a nuke is launched. The game is otherwise great, even without sound. 

Forum post:

[https://forum.faforever.com/topic/5611/request-suggestion-add-strategic-launch-detection-on-screen-notify/11
](url)

_This feature request will be read by people that are unable to look into your thoughts - this request is all they have to determine the viability of the feature. The more precise your request, the easier it becomes for us to help you._

**Comment by Garanas** [comment]
A similar 'on-screen' visual indicator would be for when your ACU is damaged. Perhaps @4z0t is interested in making this into a new UI mod?

**Comment by 4z0t** [comment]
@Garanas https://media.tenor.com/eHnJ-4ewBeEAAAAM/what-are-you-challenging-me.gif

**Comment by Garanas** [comment]
Yes, but it shouldn't be that hard. The sounds are called by the sim with:

- https://github.com/FAForever/fa/blob/deploy/fafdevelop/lua/aibrain.lua#L1462-L1497

Which just passes the information to play a sound via the sync. The UI mod would then just listen to that event and create some UI elements accordingly.

**Comment by Garanas** [comment]
Or we make it a new event of some kind, in general. I'm open to suggestions.
